src/database/firebase_object_factory.spec.ts

```diff
@@ -81,12 +81,12 @@ describe('FirebaseObjectFactory', () => {
 
 
     it('should emit unwrapped data by default', (done: any) => {
-      ref.set({ data: 'bar' }, () => {
+      const fixtureData = { data: 'bar' };
+      ref.set(fixtureData, () => {
         subscription = observable.subscribe(unwrapped => {
```

Comment:
The observable subscription shouldn't be needed here. Can you remove it and just use the inner code?